doc/api/errors.md

```diff
@@ -222,7 +222,8 @@ myObject.stack;  // similar to `new Error().stack`
 ```
 
 The first line of the trace, instead of being prefixed with `ErrorType:
-message`, will be the result of calling `targetObject.toString()`.
+message`, will be formatted as `Error: <targetObject message>` when
```

Comment:
This change results in the following text:

> The first line of the trace, instead of being prefixed with `ErrorType: message`, will be formatted as `Error: <targetObject message>`...

That seems to be saying "Instead of being FOO, it will be formatted as FOO."